src/routes/bookRoutes.ts

```diff
@@ -0,0 +1,165 @@
+import { Router } from "express";
+import {
+  getAllBooks,
+  getBook,
+  createBook,
+  deleteBook,
+} from "../controllers/bookController";
+
+// Create a router instance
+const router = Router();
+
+// Get all books
+/**
+ * @swagger
+ * /api/v1/books:
+ *   get:
+ *     summary: Get all books
+ *     tags: [Books]
+ *     responses:
+ *       200:
+ *         description: A list of all books
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: array
+ *               items:
+ *                 type: object
```

Comment:
Is there a way to specify this schema in terms of an object (i.e. an array of Book objects) so that you don't have to write the whole set of properties for a book over every route? I think it be easier to maintain if and when there are changes as well as easier to read.